Druid/DRBytecodeIRGenerator.class.st

```diff
@@ -220,11 +266,28 @@ DRBytecodeIRGenerator >> interpretNormalSpecialSelectorSendWith: aMessage [
 	
 	aMessage arguments second acceptVisitor: self.
 	argumentCount := self popOperand.
-	
+
+	self
+		addInstructionFrom: aMessage
+		instructionKind: DRAdd
+		operands: { selectorIndex . 1 asDRValue }.
+	self
+		addInstructionFrom: aMessage
+		instructionKind: DRNegate
+		operands: { self popOperand }.
```

Comment:
Why this?

Reply:
I moved the selector index calculation to the IR.
In the simple cases it was always a constant.
But in the extended bytecodes it is an expression, so I moved it to the IR to let the compiler do its thing.